### DISCUSSION TITLE
[Locked] How can I build macOS images

### DISCUSSION THREAD

**Discussion by joyhui** [actions/virtual-environments#2626] [Installed software]
The instruction only support windows or linux images.
https://github.com/actions/virtual-environments/blob/main/help/CreateImageAndAzureResources.md

GenerateResourcesAndImage -SubscriptionId {YourSubscriptionId} -ResourceGroupName "myTestResourceGroup" -ImageGenerationRepositoryRoot "$pwd" -ImageType **Ubuntu1604** -AzureLocation "East US"

How about macOs images?

**Comment by catthehacker** [comment]
See https://github.com/actions/virtual-environments/issues/1783#issuecomment-706667062

**Comment by AlenaSviridenko** [comment]
locking discussion since the question was answered
